app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatasetInterface.kt

```diff
@@ -50,6 +50,22 @@ class FirestoreDatasetInterface(
         }
     }
 
+    override suspend fun getRepresentativePicture(category: Category): CategorizedPicture {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun getAllPictures(category: Category): Set<CategorizedPicture> {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun removeCategory(category: Category) {
+        TODO("Not yet implemented")
+    }
```

Comment:
When we delete a Category in a dataset, it will also "delete" (more like remove from this dataset than really remove tho) all the pictures that belong to this category in the dataset?

Reply:
It is more of an "unlink". The category will be kept. See https://github.com/HumanLearning2021/HumanLearningApp/pull/50#discussion_r598807381